src/app/data-explorer/data-specification.service.spec.ts

```diff
@@ -484,12 +484,7 @@ describe('DataSpecificationService', () => {
         return cold('-a|', { a: undefined });
       });
 
-      submissionSDEServiceStub.mapToDataSpecification.mockImplementation((dataModel) => {
-        expect(dataModel.id).toBe(dataSpecification.id);
-        return dataSpecification;
-      });
```

Comment:
Not clear as to why this has been removed from the test?

Reply:
Neither..! Added back.